tests/README.md

```diff
@@ -47,6 +47,23 @@ MOONBEAM_LOG="warn,rpc=trace" npm run test
 The test script will find available ports above 20000 in order to ensure that it doesn't conflict
 with any other running services.
 
+## Analyzer mode
+
+The node supports being run with `perf` or with `valgrind`.  
+However it requires to run only 1 test at a time
```

Comment:
What is the behavior otherwise? I've often found that I end up matching more tests than I intend to with `--grep`, so hopefully the behavior is at least obvious.

Reply:
Well, perf will generate 1 single file matching the last test
valgrind will generate 1 file per test file probably matching the last test executed in that test file